app/views/people/show.html.erb

```diff
@@ -1,7 +1,8 @@
 <% content_for :sidebar do %>
   <%= link_to 'Return to Listing', session[:return_to], :class => 'btn btn-mini' %></br></br>
   <%= link_to 'Edit Person', edit_person_path(@person), :class => 'btn btn-mini' if can? :edit, @person  %></br></br>
-  <%= link_to 'New Certification', new_person_cert_path(@person), :class => 'btn btn-mini' if can? :create, Cert  %>
+  <%= link_to 'New Certification', new_person_cert_path(@person), :class => 'btn btn-mini' if can? :create, Cert  %></br></br>
+  <%= image_tag @person.portrait_url if @person.portrait.file.present? %>
```

Comment:
Should we have an option to follow a link to a full-size image here? In a large browser window, I'm showing a thumbnail-sized picture in the sidebar, while in a smaller window or on an iPhone I see a full picture.

Large Window:
![person display - browser large](https://cloud.githubusercontent.com/assets/8586431/17984535/9a89b80a-6adf-11e6-99ed-6e5b13b2acc3.png)

Smaller Window:
![person display - browser small](https://cloud.githubusercontent.com/assets/8586431/17984536/9a8a0648-6adf-11e6-9279-3d90bbbdfbd1.png)

iPhone:

![person display - iphone](https://cloud.githubusercontent.com/assets/8586431/17985018/eaa19d2e-6ae1-11e6-8895-45986c6b09bd.png)